apps/native/app/src/screens/more/more.tsx

```diff
@@ -130,6 +133,19 @@ export const MoreScreen: NavigationFunctionComponent = ({ componentId }) => {
               }
             />
           )}
+          {showAirDiscount && (
+            <ListButton
+              title={intl.formatMessage({ id: 'profile.airDiscount' })}
+              onPress={() => navigateTo(`/air-discount`)}
+              icon={
+                <Image
+                  source={airplaneIcon as any}
```

Comment:
I am pretty sure that any is not needed here. If you can skip then please remove the others as well :)

Reply:
Removed it and my editor was fine and looks the same when running the app, so removing it everywhere :)